docker/template/Makefile

```diff
@@ -53,6 +53,10 @@ setup:
 					EOF
 				}
 		else
+		    if [ -n "$${AWS_SESSION_TOKEN}" ]; then
+				echo "AWS_SESSION_TOKEN=$${AWS_SESSION_TOKEN}" >> $(TEMP_ENV_FILE)
+				echo "AWS_SECURITY_TOKEN=$${AWS_SECURITY_TOKEN}" >> $(TEMP_ENV_FILE)
```

Comment:
There may well be tools out there that don't set AWS_SECURITY_TOKEN, so I think the safe way is to set it from AWS_SESSION_TOKEN. See https://github.com/99designs/aws-vault/blob/33b7337ae4123fce8e9b14c587276b4034fd43dc/cli/exec.go#L273 for example.